src/private/rapidxml_formatter.cpp

```diff
@@ -431,7 +431,12 @@ namespace adm {
       node.addAttribute<AudioBlockFormatId>(&audioBlock, "audioBlockFormatID");
       node.addOptionalAttribute<Rtime>(&audioBlock, "rtime");
       node.addOptionalAttribute<Duration>(&audioBlock, "duration");
-      // TODO: add missing hoa attributes and elements
+      node.addOptionalElement<Order>(&audioBlock, "order");
+      node.addOptionalElement<Degree>(&audioBlock, "degree");
+      node.addOptionalElement<NfcRefDist>(&audioBlock, "nfcRefDist");
+      node.addOptionalElement<ScreenRef>(&audioBlock, "screenRef");
+      // TODO: add Equation
+      // TODO: add Normalization
```

Comment:
Why is the normalisation added to the class but not written by the xml writer?

Reply:
This is just a miss by me. Normalization wasn't implemented at first, so I will just have forgotten to add it. Thanks for highlighting!